action.yml

```diff
@@ -7,13 +7,36 @@ branding:
   color: 'yellow'
 
 inputs:
+  provider:
```

Comment:
📝 Consider adding a note on why the default provider is set to 'openai' for clarity.